### PR TITLE
chore: remove stale code and improve logic

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**

Refactoring. 
Removed the stale imports, unused code & improved logic.

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
NA
**Summary**
The motivation for this change -

Screenshot from 2020-03-18 17-34-51

**Does this PR introduce a breaking change?**

NO
**Other information**

### CODE REVIEW

**Comment by snitin315** [comment]
> Would suggest keeping this open for a while, go through the code and remove all stale code and logic improvement at once.

@anshumanv There are two more warnings in `/package-utils/__tests__`  regarding the use of type `any` -

Screenshot from 2020-03-18 17-49-34

I'm not sure what should be used instead of `any`.

**Comment by snitin315** [comment]
Rest all code looks good to me.

**Comment by anshumanv** [comment]
> @anshumanv There are two more warnings in /package-utils/__tests__ regarding the use of type any -

@snitin315 it should be `jest.Mock` 👍 

Let me know if that works.

**Comment by snitin315** [comment]
> Let me know if that works.

It worked thanks.

**Comment by anshumanv** [comment]
Will take a look locally in a while.

File: packages/info/src/index.ts [hunk 1]
```diff
@@ -4,7 +4,6 @@ import process from 'process';
 import { argv } from './options';
 
 import { AVAILABLE_COMMANDS, AVAILABLE_FORMATS, IGNORE_FLAGS } from './commands';
-import { configReader, fetchConfig, resolveFilePath, getNameFromPath } from './configParser';
```
Comment (anshumanv) [inline packages/info/src/index.ts]: This file is also unused, can be safely removed. Not really sure why it was introduced, @rishabh3112 any ideas?

Comment (snitin315) [reply packages/info/src/index.ts]: @anshumanv  this file is being used in some test suites. , 

Screenshot from 2020-03-22 10-59-57

Comment (anshumanv) [reply packages/info/src/index.ts]: I think you forogt to rebuild, passing fine for me after removing the file.

Comment (anshumanv) [reply packages/info/src/index.ts]: And no, not used, the fn you showed is in `index.ts` in info, not in `configParser`.

Comment (snitin315) [reply packages/info/src/index.ts]: Accidently removed some extra code. My bad ;-)

**Comment by jamesgeorge007** [comment]
Thanks

**Pull request merged**